edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesRuntimeInfoProvider.cs

```diff
@@ -128,12 +126,12 @@ public async Task<SystemInfo> GetSystemInfo()
         async Task WatchPodEventsAsync(WatchEventType type, V1Pod item)
         {
             // if the pod doesn't have the module label set then we are not interested in it
-            if (! item.Metadata.Labels.ContainsKey(Constants.k8sEdgeModuleLabel))
+            if (!item.Metadata.Labels.ContainsKey(Constants.K8sEdgeModuleLabel))
```

Comment:
nit: Don't need to lookup dictionary twice
```
if (!item.TryGetValue(Constants.K8sEdgeModuleLabel, out var podName))
    return;
....
```

Reply:
Agreed, done.